docs/en/concepts-and-designs/project-goals.md

```diff
@@ -0,0 +1,14 @@
+# Design Goals
+The document outlines the core design goals for the SkyWalking Infra E2E project.
+
+- Keep the feature parity with the existing E2E framework in SkyWalking main repo.
```

Comment:
```suggestion
- Support various E2e testing requirements in SkyWalking main repository with other ecosystem repositories.
```

Reply:
The current e2e testing is not available any more after we finish the replacement.